kornia/geometry/transform/affwarp.py

```diff
@@ -150,6 +224,42 @@ def rotate(tensor: torch.Tensor, angle: torch.Tensor,
     return affine(tensor, rotation_matrix[..., :2, :3], mode, align_corners)
 
 
+def rotate3d(tensor: torch.Tensor, yaw: torch.Tensor, pitch: torch.Tensor, roll: torch.Tensor,
```

Comment:
Wouldn't be clean to pass *,3 tensor with angles? @ducha-aiki

Reply:
I don't have strong opinion on it. Probably, 3 with explicit names is easier to understand